workspaces/redhat-argocd/plugins/argocd/src/components/DeploymentLifeCycle/sidebar/resources/ResourcesTable.tsx

```diff
@@ -69,24 +74,41 @@ export const ResourcesTable: FC<ResourcesTableProps> = ({
     [resources],
   );
 
+  const getAllKinds = React.useMemo(() => {
+    return resources.reduce((kinds: string[], resource) => {
+      if (resource.kind) {
+        kinds.push(resource.kind);
+      }
```

Comment:
![image](https://github.com/user-attachments/assets/64197099-08b8-44fb-a8b2-9b76769e23f4)


Need a check to avoid multiple same kinds
```suggestion
      if (resource.kind && !kinds.includes(resource.kind)) {
        kinds.push(resource.kind);
      }
```